src/utils/openCloseComponent.ts

```diff
@@ -60,6 +61,26 @@ function transitionEnd(event: TransitionEvent): void {
     this.open ? this.onOpen() : this.onClose();
   }
 }
+
+/**
+ * Helper to determine globally set transition duration setting.
+ *
+ * @param component
+ */
+export function onToggleUnanimatedComponent(component: OpenCloseComponent): void {
+  readTask((): void => {
+    if (component.transitionEl) {
```

Comment:
What's the scenario where this method would be called and `transitionEl` is not available? And how does the method get called again once `transitionEl` is available?